concepts/0046-mediators-and-relays/README.md

```diff
@@ -0,0 +1,242 @@
+# 0036: Mediators and Relays
```

Comment:
The RFC number here does not match the number in the folder name.

Reply:
Thank you for the catch, @troyronda ! I will update.

Reply:
Cool. I had already created https://github.com/hyperledger/aries-rfcs/pull/57 if you want to use it.